lib/datasource/helm/index.ts

```diff
@@ -62,9 +62,10 @@ export class HelmDatasource extends Datasource {
       }
       const result: HelmRepositoryData = {};
       for (const [name, releases] of Object.entries(doc.entries)) {
-        const { sourceUrl, sourceDirectory } = findSourceUrl(releases[0]);
+        const refRelease = releases[releases.length - 1];
```

Comment:
Are you sure the latest release is the last? I'm pretty sure I often saw the opposite. So releases should be sorted with semver.

Reply:
you're right. It looks like a Sonatype Nexus specific issue then.